docs/contracts/v2/guides/proxy-architecture/_category_.json

```diff
@@ -0,0 +1,5 @@
+{
+  "collapsed": true,
+  "label": "Proxy Architecture",
+  "position": 3
```

Comment:
Should be 4, and the position for other pages in the side bar should be updated, too. E.g. 04-hooks becomes 05-hooks, and its position is changed to 5.